### DISCUSSION TITLE
[PSD1] Exercise 1.11

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#116] [Esercizi (A-L)]
**Exercise 1.11**

What is the smallest (most negative) 16-bit binary number that can be represented with:
	(a) unsigned numbers?
	(b) two’s complement numbers?
	(c) sign/magnitude numbers?

**Comment by Elia-Belli** [comment]
Il più piccolo numero binario a 16bit rappresentabile con le seguenti codifiche è:

- Unsigned: $0$
- Complemento a 2:  $-2^{15}$
- Sign/magnitude: $-2^{15}+1$
